examples/add-gitignore.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# run this script from examples folder
+# will copy .gitignore file to all example folders 
+# it will not overwrite an existing .gitignore file
+# -n option means no overwrite
+# -v option means verbose
+
+for folder in * ; 
+  do cp -n -v .gitignore $folder; 
+done;
```

Comment:
We don't need this anymore so let's remove this file too

Reply:
@lfades, 
It was mentioned by @Timer in #15102 , that a script is required.

Reply:
@tanmaylaud This script is used to add `.gitignore` to examples missing it, not to update them, new examples won't get merged unless they have a `.gitignore` so this script won't be needed.

A custom script to update the ignored files may be useful, but as of now I don't think that it will be needed. cc @Timer if needed, we can add something in the future.

Reply:
The script should stay, we need it!

Reply:
@Timer, @lfades : can we reach a consensus?